From: "M.Angel Juan" <m.angel.juan@gmail.com>
Date: Wed, 27 Jul 2022 10:51:46 +0200
Subject: Add ca language to desktop file

---
 data/com.github.jeromerobert.pdfarranger.desktop | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/com.github.jeromerobert.pdfarranger.desktop b/data/com.github.jeromerobert.pdfarranger.desktop
index b96fee8..c902a9e 100644
--- a/data/com.github.jeromerobert.pdfarranger.desktop
+++ b/data/com.github.jeromerobert.pdfarranger.desktop
@@ -4,6 +4,8 @@ Name=PDF Arranger
 Name[hr]=PDF uređivač
 Name[hu]=PDF-rendező
 Name[ko]=PDF 편집기
+Name[es]=Mezclador PDF
+Name[ca]=Mesclador PDF
 Comment=PDF Merging, Rearranging, Splitting, Rotating and Cropping
 Comment[cs]=Program pro sloučení, přeuspořádání, rozdělení, otočení a oříznutí PDF
 Comment[de]=PDFs zusammenfügen, umsortieren, trennen, rotieren und zuschneiden
@@ -15,6 +17,7 @@ Comment[sk]=Grafický nástroj na manipuláciu s PDF
 Comment[sv]=Slå samman, arrangera om, dela, rotera och beskär PDF filer
 Comment[fi]=Yhdistä, järjestele, jaa, käännä ja rajaa PDF-tiedostoja
 Comment[ko]=PDF 합치기, 재정렬, 나누기, 회전 및 잘라내기
+Comment[ca]=Mescle, reordene, dividisca, rote i retalle documents PDF
 Type=Application
 Exec=pdfarranger %U
 Icon=com.github.jeromerobert.pdfarranger
